auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtension.java

```diff
@@ -214,6 +215,7 @@ FieldSpec generateCreator(ProcessingEnvironment env, List<Property> properties,
 
     MethodSpec.Builder createFromParcel = MethodSpec.methodBuilder("createFromParcel")
         .addAnnotation(Override.class)
+        .addAnnotation(createSuppressUncheckedWarningAnnotation())
```

Comment:
@JakeWharton, I can add that logic but does it worth the bother and chance of bugs?